transport/wseb/src/main/java/org/kaazing/gateway/transport/wseb/WsebAcceptor.java

```diff
@@ -107,7 +111,7 @@
 import org.kaazing.mina.netty.IoSessionIdleTracker;
 import org.kaazing.mina.netty.util.threadlocal.VicariousThreadLocal;
 
-// TODO: will need some sort of session cleanup timeout
+@SuppressWarnings("deprecation")
```

Comment:
What's the class that triggers the deprecation warning when this is removed?
Perhaps we could move this closer to the call site so that other calls to deprecated methods in this class are not obscured accidentally?